pegasus/sites.v3/code.org/views/teacher_announcements.haml

```diff
@@ -7,10 +7,10 @@
         >>
   %div.announce-content
     %div.content-img-container
-      %img.content-img{:src=>"/images/mc/fit-x108/MC_Box_V3.jpg"}
+      %img.content-img{:src=>"/images/fit-x108/volunteer-engineer-thumbnail.jpeg"}
     %div.content-text
-      %b November 16, 2015 -
-      A new Hour of Code tutorial is here, featuring the popular game, Minecraft. Use blocks of code to take Steve or Alex on an adventure through this Minecraft world.
+      %b November 23, 2015 -
+      Would you like a technical volunteer in your classroom for the Hour of Code? Check out the volunteer map and get in contact with volunteers in your area.
```

Comment:
Would you like a volunteer passionate about computer science education to help during your Hour of Code event? Over 7,000 volunteers are looking to visit your classroom or video chat remotely to inspire your students. Check out the volunteer map to contact volunteers in your area.

And we should localize this, add as a string in i18n gsheet like the learn more button.